app/assets/javascripts/active_admin/lib/has_many.js.coffee

```diff
@@ -52,9 +52,10 @@ $ ->
   $(document).on 'change','.has_many_container[data-sortable] :input[name$="[_destroy]"]', ->
     recompute_positions $(@).closest '.has_many'
 
-  init_sortable()
   $(document).on 'has_many_add:after', '.has_many_container', init_sortable
 
+$(document).on 'ready page:load', ->
```

Comment:
On Rails 5, `page:load` becomes `turbolinks:load`. I believe it would be better to wrap the document ready closure in a function and call it like so:

``` coffeescript
ready_hook = ->
  # current content of $ ->

$(document).on 'ready', ready_hook
$(document).on 'page:load', ready_hook
$(document).on 'turbolinks:load', ready_hook
```

This *should* cover all cases, but I haven't tested it.